### PR TITLE
fixes spacing/orientation xform ordering

### PR DESCRIPTION
**Pull request by wyli**
Signed-off-by: Wenqi Li <wenqil@nvidia.com>

part of https://github.com/Project-MONAI/tutorials/issues/545

### Status
**Ready**

### Types of changes

- [x] Non-breaking change (fix or new feature that would not break existing functionality).
- [ ] Breaking change (fix or new feature that would cause existing functionality to change).
- [ ] New tests added to cover the changes.
- [ ] Integration tests passed locally by running `./runtests.sh -f -u --net --coverage`.
- [ ] Quick tests passed locally by running `./runtests.sh --quick --unittests  --disttests`.
- [ ] In-line docstrings updated.
- [ ] Documentation updated, tested `make html` command in the `docs/` folder.

### CODE REVIEW

File: monai/apps/deepgrow/dataset.py [hunk 1]
```diff
@@ -126,8 +126,8 @@ def _default_transforms(image_key, label_key, pixdim):
         [
             LoadImaged(keys=keys),
             AsChannelFirstd(keys=keys),
-            Spacingd(keys=keys, pixdim=pixdim, mode=mode),
```
Comment (wyli) [inline monai/apps/deepgrow/dataset.py:129]: the orientation should happen first, could you please help evaluate the impact of this change? most likely we don't need to retrain any existing models... @SachidanandAlle @finalelement @diazandr3s.

Comment (SachidanandAlle) [reply monai/apps/deepgrow/dataset.py:129]: yes.. this should be ok.. we change the orientation while training the model..  for existing pre-trained models.. this should not impact..

Comment (wyli) [reply monai/apps/deepgrow/dataset.py:129]: ok, that's right thanks for confirming it.

Comment (diazandr3s) [reply monai/apps/deepgrow/dataset.py:129]: Agree with Sachi. We are already using DeepEdit without the Spacing transform :)

**Review by holgerroth — APPROVED** [review]
LGTM

**Pull request merged**